packages/lms/src/lib/components/navigation/ProgrammeNav.svelte

```diff
@@ -44,42 +44,20 @@
 		</h2>
 		<div class="inst-grid">
 			{#each current.children as child}
-				<div class="card">
-					<h3>{child.title}</h3>
-					<img src={child.url} alt={child.title} />
-					<p>{child.summary}</p>
-					<a href={child.browserPath}>View Details</a>
+				<div class="block card p-4">
+					<h3 class="card-header">{child.title}</h3>
+					{#if child.url}
+						<img src={child.url} alt={child.title} />
+					{/if}
+					<p class="p-4">{child.summary}</p>
+					<a class="card-footer btn variant-filled bg-primary-500" href={child.browserPath}>View Details</a>
 				</div>
```

Comment:
```suggestion
				<a class="card p-4 variant-ghost-surface" href={child.browserPath}>
					<header class="card-header border-b pb-2"><h3 class="h3">{child.title}</h3></header>
					<section>
						{#if child.url}
							<img src={child.url} alt={child.title} />
						{/if}
						<p class="p-4">{child.summary}</p>
					</section>
				</a>
```

This style matches the one I chose for cards on the search page. We should talk and get to an agreement on how to style these for consistency purposes. I attach pics of what they look like on the search page. Bear in mind that I changed the surface colour in the theme because the previous one was too dark, so if you apply these suggested changes you won't see the same colours that the pictures show you until you've merged in `skeleton-styling`
![cards light mode](https://github.com/Fermain/-mollify/assets/76968122/ac15af7b-5f0d-4799-93c7-184223413fb1)
![cards dark mode](https://github.com/Fermain/-mollify/assets/76968122/942564b4-2065-482d-ac7c-341d2984687a)